routes/routes.go

```diff
@@ -19,7 +13,26 @@ import (
 // Create is used to create a new router
 func Create(scheduler types.Scheduler, conf *config.Config) *mux.Router {
 	h := handler.Create(scheduler, conf.Database)
-	routes := types.Routes{
+	router := mux.NewRouter().StrictSlash(true)
+
+	for _, route := range routes(h, conf) {
+		router.
+			Methods(route.Method).
+			Path(route.Pattern).
+			Name(route.Name).
+			Handler(prometheus.InstrumentHandler(route.Name, route.Handler))
+	}
+
+	router.PathPrefix("/static/").
```

Comment:
The "mixed" indentation looks a bit wonky. I would prefer to move the first method to a new line like we do in the loop above.